source/developers-guide/debugging/index.md

```diff
@@ -40,18 +47,22 @@ array(
     'phpsettings' => [
         'display_errors' => 1,
     ],
-)
+]
 ```
 
 ## PHP
 
 ### Xdebug
 
-Xdebug is a very common and convenient way to debug your PHP applications. It will allow you to debug a request step by step and inspect variables and object values at any point.
+Xdebug is a very common and convenient way to debug your PHP applications.
+It will allow you to debug a request step by step and inspect variables and object values at any point.
 
-It can be found in all common linux distributions, e.g. in Ubuntu as `php5-xdebug`. After installing the extension, you will need to configure the xdebug php extension, e.g. in the file `/etc/php5/apache2/conf.d/20-xdebug.ini` (this might vary depending on your distribution and PHP setup). Using a local setup, your configuration might look like this:
+It can be found in all common linux distributions, e.g. in Ubuntu as `php5-xdebug`.
+After installing the extension, you will need to configure the xdebug php extension,
+e.g. in the file `/etc/php5/apache2/conf.d/20-xdebug.ini` (this might vary depending on your distribution and PHP setup).
```

Comment:
```suggestion
e.g. in the file `/etc/php7/apache2/conf.d/20-xdebug.ini` (this might vary depending on your distribution and PHP setup).
```

Reply:
I don't know whether those paths are correct, just so we don't have references to PHP5 in the documentation anymore ^^